specs/simple-serialize.md

```diff
@@ -107,7 +107,7 @@ variable_parts = [serialize(element) if is_variable_size(element) else b"" for e
 
 # Compute and check lengths
 fixed_lengths = [len(part) if part != None else BYTES_PER_LENGTH_OFFSET for part in fixed_parts]
-variable_lengths = [len(part) for part in variable_parts]
+variable_lengths = [len(part) if len(part) != 0 for part in variable_parts]
```

Comment:
Have you tested this? It looks like it breaks the variable_offsets calculation (down a few lines):

```python
variable_offsets = [serialize(sum(fixed_lengths + variable_lengths[:i])) for i in range(len(value))]
fixed_parts = [part if part != None else variable_offsets[i] for i, part in enumerate(fixed_parts)]
```

There needs to be a length for every item, even if 0.